taca/backup/backup.py

```diff
@@ -226,20 +231,30 @@ def _log_pdc_statusdb(self, run):
 
     def _is_ready_to_archive(self, run, ext):
         """Check if the run to be encrypted has finished sequencing and has been copied completely to nas"""
-
         archive_ready = False
-
         run_path = run.abs_path
         rta_file = os.path.join(run_path, self.finished_run_indicator)
         cp_file = os.path.join(run_path, self.copy_complete_indicator)
-        if os.path.exists(rta_file) and os.path.exists(cp_file) and (not self.file_in_pdc(run.zip_encrypted)):
+        if self._get_run_type(run.name) == 'promethion' or self._get_run_type(run.name) == 'minion':
+            if os.path.exists("/".join([run_path, ".sync_finished"])):
+                # Case for encrypting
+                # Run has NOT been encrypted (run.tar.gz.gpg not exists)
+                if ext == '.tar.gz' and (not os.path.exists(run.zip_encrypted)):
+                    logger.info(f'Sequencing has finished and copying completed for run {os.path.basename(run_path)} and is ready for archiving')
+                    archive_ready = True
+                # Case for putting data to PDC
+                # Run has already been encrypted (run.tar.gz.gpg exists
+                elif ext == '.tar.gz.gpg' and os.path.exists(run.zip_encrypted):
+                    logger.info(f'Sequencing has finished and copying completed for run {os.path.basename(run_path)} and is ready for sending to PDC')
+                    archive_ready = True
+        elif os.path.exists(rta_file) and os.path.exists(cp_file) and (not self.file_in_pdc(run.zip_encrypted)):
             # Case for encrypting
             # Run has NOT been encrypted (run.tar.gz.gpg not exists)
             if ext == '.tar.gz' and (not os.path.exists(run.zip_encrypted)):
                 logger.info(f'Sequencing has finished and copying completed for run {os.path.basename(run_path)} and is ready for archiving')
                 archive_ready = True
             # Case for putting data to PDC
-            # Run has already been encrypted (run.tar.gz.gpg exists)
+            # Run has already been encrypted (run.tar.gz.gpg exists
```

Comment:
A missing `)`?

Reply:
yeah, it was removed by mistake :/, put it back